pkgs/tools/networking/dnsmasq/default.nix

```diff
@@ -39,6 +39,11 @@ stdenv.mkDerivation rec {
   # module can create it in Nix-land?
   postInstall = ''
     install -Dm644 trust-anchors.conf $out/share/dnsmasq/trust-anchors.conf
+  '' + optionalString stdenv.isDarwin ''
+    install -Dm644 contrib/MacOSX-launchd/uk.org.thekelleys.dnsmasq.plist \
+      $out/System/Library/LaunchDaemons/uk.org.thekelleys.dnsmasq.plist
```

Comment:
Huh, you're right, I found two that use `Library` and only one that used `System/Library`. Guess I didn't look closely enough the first time.  I'll change my patch to match the majority.